### PR TITLE
fix(docs): remove double words

### PR DESCRIPTION
**Pull request by muescha**
## Description

changes:

- remove double words

## Note

leaving double words like:
- very very fast
- very very well-documented
- please please
- far far away
- much much more
- much much faster
- Hacktoberfest Hacktoberfest Hacktoberfest
- and some useless lorem ipsum double words

### CODE REVIEW

File: docs/blog/2018-02-16-bright-future-for-the-web/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ It was for those reasons that I was hesitant to pull the trigger on a static sit
...
+Next came integrating the statically generated site with data stored in a CMS. Again Gatsby was well suited for this and I was able to easily integrate with Contentful via the [gatsby-source-contentful](/packages/gatsby-source-contentful/) plugin. Contentful is an example of a [headless CMS](/docs/headless-cms/), meaning that is not tied to any particular technology or language. Contentful allows you to store content using a pleasant user interface and it can then output the data via an API. Best of all Contentful puts you in the driver seat and lets you define your own content model however you see fit. Think [WordPress Advanced Custom Fields](https://www.advancedcustomfields.com/) on steroids. With the content stored in Contentful Gatsby then uses the Contentful API along with the awesome power of [GraphQL](https://graphql.org/) to query data at build time. Cool stuff!
```
Comment (muescha) [inline docs/blog/2018-02-16-bright-future-for-the-web/index.md]: meaning that is **is** not tied

File: docs/blog/2019-02-07-pragmatic-lessons-from-converting-to-react-hooks/index.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ const updateStateGood = () =>
...
+The custom `useSlideshow` hook utilizes two different hooks to replace the functionality of the render props component: `useState` and `useEffect`. The current index and playing states are both set with their own calls to useState. The `useEffect` hook checks if the isPlaying state is true and then sets the timeout to advance the slide to the next index. It resets to the first slide after it reaches the last index. The hook clears the timeout when the current index or isPlaying state changes. The hook includes a function to update the slide. The necessary state and functions are return in an object.
```
Comment (muescha) [inline docs/blog/2019-02-07-pragmatic-lessons-from-converting-to-react-hooks/index.md]: function to update the **the** slide

File: docs/blog/2018-02-16-bright-future-for-the-web/index.md (hunk 1 above)
Comment (laurieontech) [reply docs/blog/2018-02-16-bright-future-for-the-web/index.md]: I believe this is meant to say "it is"

File: docs/blog/2018-02-16-bright-future-for-the-web/index.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ It was for those reasons that I was hesitant to pull the trigger on a static sit
...
+Next came integrating the statically generated site with data stored in a CMS. Again Gatsby was well suited for this and I was able to easily integrate with Contentful via the [gatsby-source-contentful](/packages/gatsby-source-contentful/) plugin. Contentful is an example of a [headless CMS](/docs/headless-cms/), meaning that it is not tied to any particular technology or language. Contentful allows you to store content using a pleasant user interface and it can then output the data via an API. Best of all Contentful puts you in the driver seat and lets you define your own content model however you see fit. Think [WordPress Advanced Custom Fields](https://www.advancedcustomfields.com/) on steroids. With the content stored in Contentful Gatsby then uses the Contentful API along with the awesome power of [GraphQL](https://graphql.org/) to query data at build time. Cool stuff!
```
Comment (muescha) [inline docs/blog/2018-02-16-bright-future-for-the-web/index.md]: changed
```diff
-meaning that is is not
+meaning that it is not
```

File: docs/blog/2018-02-16-bright-future-for-the-web/index.md (hunk 1 above)
Comment (muescha) [reply docs/blog/2018-02-16-bright-future-for-the-web/index.md]: changed :)

**Comment by muescha** [comment]
@laurieontech changed to `it is`

**Comment by laurieontech** [comment]
Thanks!

**Pull request merged**